src/main/java/com/google/sps/servlets/KeywordServlet.java

```diff
@@ -56,5 +88,21 @@ private String jsonBuilder(Collection<String> toJson) {
   private String keyValueJson(String key, String value) {
```

Comment:
If this function is only one line, it may be better to just put this code in jsonBuilder().